package.json

```diff
@@ -32,10 +32,11 @@
     }
   ],
   "scripts": {
+    "lint": "eslint .",
     "test": "npx jasmine-browser-runner runSpecs",
     "build": "rollup --config rollup.config.ts --configPlugin @rollup/plugin-typescript",
```

Comment:
Can we make `test` task as `npm run jasmine-browser-runner runSpecs` because we already have local dependency in the project. `npx` uses global one so you need to install it the second time, but why?